scripts/predict.py

```diff
@@ -1,45 +1,56 @@
+from __future__ import print_function
 import json
 import os
+import re
 import pickle
 import sys
+import click
 from os.path import dirname, abspath
 sys.path.append(dirname(dirname(abspath(__file__))))
 from pkspace.utils.loaders import PKSpaceLoader # noqa
-import click # noqa
 
 
 @click.command()
-@click.option('--mask_path', required=True,
-              help='Adress of json file of the mask')
-@click.option('--picture_path', required=True,
-              help='Adress of picture to be predicted')
-@click.option('--model_path', required=True,
-              help='Path where pretrained model')
+@click.option('--mask_path', required=True, type=click.Path(exists=True),
+              help='Path of json file of the mask')
+@click.option('--picture_path', required=True, type=click.Path(exists=True),
+              help='Path of picture to be predicted')
+@click.option('--model_path', required=True, type=click.Path(exists=True),
```

Comment:
@Adman as per http://click.pocoo.org/5/api/#click.Path I believe it would make more sense to check for `file_okay` and also add `resolve_path=True`.